jenkins/spark-tests.sh

```diff
@@ -211,6 +211,20 @@ run_iceberg_tests() {
   fi
 }
 
+ # Test spark-avro with documented way of deploying at run time via --packages option from Maven
+run_avro_tests() {
+  export PYSP_TEST_spark_jars_packages="org.apache.spark:spark-avro_2.12:${SPARK_VER}"
+
+  # Workaround to avoid appending avro jar file by '--jars'
+  AVRO_JAR_FILE=`cd $LOCAL_JAR_PATH && ls spark-avro*.jar`
+  mv $LOCAL_JAR_PATH/$AVRO_JAR_FILE $LOCAL_JAR_PATH/$AVRO_JAR_FILE%
```

Comment:
one thing that could cause issue is that mv could fail if no avro file in LOCAL_JAR_PATH. 
may simply `|| true` to avoid case that people mark ENV `INCLUDE_SPARK_AVRO_JAR=false`

Reply:
reverted to previous simple workaround. It would be fine as avro test would be executed at the end of scripts and avro jar file will be auto downloaded each time